lua/spider/precognition-integration/init.lua

```diff
@@ -0,0 +1,29 @@
+local M = {}
+--- Register Precognition Spider Motions Adapter
+M.register_adapter = function()
+	local globalOpts = require("spider.config").globalOpts
+	local ok, vanillaMotions = pcall(require, "precognition.motions.vanilla_motions")
+	if not ok then
+		vim.api.nvim_echo({ { "[Spider] `precognition` not found" } }, false, { err = false })
```

Comment:
since `register_adapter` is called in spider's `setup`, this will result in every user who does not have `precognition` installed to get the "precognition not found" message on every startup.

Think you could just remove the line; if the user does not have precognition installed, simply return early  silently.